tests/Functional/ProxyGeneratorTest.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+namespace Zenstruck\Foundry\Tests\Functional;
+
+use Zenstruck\Foundry\AnonymousFactory;
+use Zenstruck\Foundry\Tests\Fixtures\Entity\Contact;
+use Zenstruck\Foundry\Tests\Fixtures\Entity\Post;
+use Zenstruck\Foundry\Tests\Fixtures\Factories\PostFactoryWithProxyGenerator;
+
+/**
+ * @author Wouter de Jong <wouter@wouterj.nl>
+ */
+final class ProxyGeneratorTest extends ProxyTest
+{
+    protected static $POST_FACTORY = PostFactoryWithProxyGenerator::class;
+
+    /**
+     * @test
+     */
+    public function can_assert_persisted(): void
+    {
+        $this->markTestSkipped('not supported');
+    }
```

Comment:
TODO: we probably have to move the `assertPersisted()` to the assertions class

Reply:
I don't personally find the Proxy assertions useful, maybe we can remove in 2.0?

Reply:
That sounds great :)

Honestly, I personally would like to remove as much custom methods from the proxy as possible. I much rather use the normal Doctrine API. But I know you're quite fond of these little helper methods. In short: you're completely at the lead as to which proxy methods to keep and which to drop in 2.0 :)